integrations/mongodb.md

```diff
@@ -16,17 +16,64 @@ logo: /logos/mongodb.png
 toc: true
 ---
 
+### Table of Contents
+
+- [Overview](#overview)
+- [Haystack 2.0](#haystack-20)
+  - [Installation](#installation)
+  - [Usage](#usage)
+- [Haystack 1.x](#haystack-1x)
+  - [Installation (1.x)](#installation-1x)
+  - [Usage (1.x)](#usage-1x)
+  - [Writing Documents to MongoDBAtlasDocumentStore](#writing-documents-to-mongodbstlasdocumentstore)
+  - [Indexing Pipeline](#indexing-pipeline)
+  - [Query Pipeline](#query-pipeline)
+
+## Overview
+
 [MongoDB](https://www.mongodb.com/) is a document database designed for ease of application development and scaling. [MongoDB Atlas](https://www.mongodb.com/atlas) is a multi-cloud database service built by people behind MongoDB. MongoDB Atlas simplifies deploying and managing your databases while offering the versatility you need to build resilient and performant global applications on the cloud providers of your choice.
 
 For a detailed overview of all the available methods and settings for the `MongoDBAtlasDocumentStore`, visit the Haystack [Documentation](https://docs.haystack.deepset.ai/docs/document_store#initialization).
 
-## Installation
+## Haystack 2.0
+
+
+### Installation
+
+```bash
+pip install mongodb-atlas-haystack
+```
+
+### Usage
+
+To use MongoDB Atlas as your data storage for your Haystack LLM pipelines, you must have a running database at MongoDB Atlas. For details, see [Get Started with Atlas](https://www.mongodb.com/docs/atlas/getting-started/).  
+
+Once your database is set, first export an environment variable called `MONGO_CONNECTION_STRING`, which should look more or less like this:
+
+```bash
+export MONGO_CONNECTION_STRING="mongodb+srv://<username>:<password>@<cluster_name>.gwkckbk.mongodb.net/?retryWrites=true&w=majority"
+```
+
+And then you can initialize a `MongoDBAtlasDocumentStore` for Haystack with required configurations:
```

Comment:
Can you add the docs links of MongoDBAtlasDocumentStore here

Reply:
Note: the link won't work until the docs are published.